src/java.base/share/classes/java/nio/charset/spi/CharsetProvider.java

```diff
@@ -36,14 +36,28 @@
  * zero-argument constructor and some number of associated charset
  * implementation classes.  Charset providers may be installed in an instance
  * of the Java platform as extensions.  Providers may also be made available by
```

Comment:
It is not related to module path, but taking this opportunity, let's amend this sentence that refers to "extensions" which is obsolete.